stdlib/public/runtime/HeapObject.cpp

```diff
@@ -61,7 +61,7 @@ using namespace swift;
 /// Returns true if the pointer passed to a native retain or release is valid.
 /// If false, the operation should immediately return.
 static inline bool isValidPointerForNativeRetain(const void *p) {
-#if defined(__x86_64__) || defined(__arm64__) || defined(__aarch64__) || defined(_M_ARM64) || defined(__s390x__)
+#if defined(__x86_64__) || defined(__arm64__) || defined(__aarch64__) || defined(_M_ARM64) || defined(__s390x__) || defined(__powerpc64__) || defined(__ppc__) || defined(__PPC__) || defined(__LITTLE_ENDIAN__)
```

Comment:
Why is `__LITTLE_ENDIAN__` being added?

Reply:
Updated the condition as suggested "|| (defined(__powerpc64__) && defined(__LITTLE_ENDIAN__))" above and tested the same. Please review commit 7dbda7847d0e99d5ed7e40c65c75535cca88217c.

Reply:
Updated the condition as suggested || (defined(__powerpc64__) && defined(__LITTLE_ENDIAN__)) above and tested the same. Please review commit 7dbda7847d0e99d5ed7e40c65c75535cca88217c

Reply:
Can this be approved and merged now? Thanks in advance!

Reply:
Hi @compnerd @mikeash , can you please review this? This PR is resolving a blocker while building swift toolchain on PowerPC64LE and is a bit urgent. Since this is a small change, can you please take this up? Thanks!

Reply:
Sorry for letting this sit. The new version looks good. I'll kick off testing and merge it upon success.

Reply:
And merged. Thanks!

Reply:
Thanks @mikeash!